src/pytest_postgresql/executor.py

```diff
@@ -136,19 +136,24 @@ def init_directory(self):
         # remove old one if exists first.
         self.clean_directory()
         init_directory = (
-            self.executable, 'initdb',
-            '-o "--auth=trust --username=%s"' % self.user,
-            '-D %s' % self.datadir,
+            self.executable, 'initdb', '-D {}'.format(self.datadir)
         )
 
         if self.password:
-            with tempfile.NamedTemporaryFile() as password_file:
+            with tempfile.NamedTemporaryFile(mode='w') as password_file:
                 init_directory += (
-                    '--pwfile "%s"' % password_file.name,
+                    '-o "--auth=trust --username={} --pwfile={}"'.format(
```

Comment:
you can have many -o outputs, add here only the pwfile

Reply:
Didn't know about that! Surely that simplifies code. Added.